src/full/Agda/TypeChecking/Unquote.hs

```diff
@@ -917,7 +916,7 @@ evalTCM v = Bench.billTo [Bench.Typing, Bench.Reflection] do
 
     constInfo :: QName -> TCM Definition
     constInfo x = either err return =<< getConstInfo' x
-      where err _ = genericError $ "Unbound name: " ++ prettyShow x
+      where err _ = typeError $ NotInScope $ qnameToConcrete x
```

Comment:
Do we want to throw this NotInScope in reflection, or should there be a variant of this?
@UlfNorell @jespercockx @plt-amy